src/intTest/java/uk/nhs/digital/nhsconnect/nhais/uat/CustomArgumentsProvider.java

```diff
@@ -33,6 +33,8 @@ public CustomArgumentsProvider(String folder) {
         var resources = getResources();
 
         var grouped = Arrays.stream(resources)
+            .filter(r -> !r.getFilename().endsWith("txt")) // ignore notes
```

Comment:
it's not clear what the purpose of the parameter r is

Reply:
ignores files that end in txt - those are notes as described in the readme